src/main/java/com/bookclub/model/Book.java

```diff
@@ -1,84 +1,44 @@
 package com.bookclub.model;
 
-/**
- * The {@code Book} class represents a book in the book club application.
- * Each book has a title and an author. This class provides getter and setter methods for these attributes.
- * Both title and author must be non-null and non-empty.
- */
 public class Book {
 
     private String title;
     private String author;
+    private Integer bookid;
+    private String genre;
 
-    /**
-     * Constructs a {@code Book} object with the specified title and author.
-     * Both title and author must be non-null and non-empty.
-     *
-     * @param title  the title of the book
-     * @param author the author of the book
-     * @throws IllegalArgumentException if the title or author is null or empty
-     */
-    public Book(String title, String author) {
-        setTitle(title);
-        setAuthor(author);
+    public Book(Integer bookid, String title, String author, String genre) {
```

Comment:
By my comment I meant for constructors like this, we don't need to worry about passing in all the content that could be optional (null) during construction as we can add these later. 

Would also be good to not have to worry about ID whenever we make something as it'll never be correct unless it comes from the DAO. So unforunately I'm asking to essentially revert the change to the constructor back to the original one.

```suggestion
    public Book(String title, String author) {
```